pro/src/screens/IndividualOffer/DetailsScreen/__specs__/DetailsScreen.spec.tsx

```diff
@@ -656,54 +662,113 @@ describe('screens:IndividualOffer::Informations', () => {
   })
 
   describe('on creation', () => {
-    it('should render EAN search for record stores as a venue', () => {
-      const context = individualOfferContextValuesFactory({
-        categories,
-        subCategories,
-        offer: null,
-      })
+    describe('about EAN search', () => {
+      const eanSearchTitle = /Scanner ou rechercher un produit par EAN/
+      const eanInputLabel = /Nouveau Scanner ou rechercher un produit par EAN/
+      const eanButtonLabel = /Rechercher/
+
+      it('should render EAN search for record stores as a venue', () => {
+        const context = individualOfferContextValuesFactory({
+          categories,
+          subCategories,
+          offer: null,
+        })
+
+        renderDetailsScreen(
+          {
+            ...props,
+            venues: [
+              venueListItemFactory({
+                venueTypeCode: 'RECORD_STORE' as VenueTypeCode,
```

Comment:
pas moyen de mettre le vrai type pour éviter le `as` ?

Reply:
J'ai répliqué ce qui existait déjà, il y a un problème sur les types issus de l'api. Voir : pro/src/screens/OfferType/__specs__/OfferType.navigation.spec.tsx.
> cast is needed because VenueTypeCode in apiClient is defined in french, but sent by api in english